install/kubernetes/helm/istio/values.yaml

```diff
@@ -180,6 +180,9 @@ global:
     includeIPRanges: "*"
     excludeIPRanges: ""
 
+    # pod internal interfaces
```

Comment:
Please add some explanation. "Traffic to/from internal interfaces will not be captured and redirected to Envoy"